### DISCUSSION TITLE
Adding product icon for Event Streams to Paste

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1028] [Request: Icon] [answered]
Hey y'all! My team is working on building out the Event Streams experience. What is the process to get our [product icon](https://www.twilio.com/brand/resources/logos) added to Paste, our sketch library, and Console.Zen? 

Attached zip folder for asset created by our Brand team: 
[event-streams.zip](https://github.com/twilio-labs/paste/files/5674259/event-streams.zip)

**Comment by TheSisb** [comment, marked as answer]
Hi @mindytom,

We've got a page on our documentation website outlining the process: https://paste.twilio.design/icons/how-to-add-an-icon.
